src/site/xdoc/contributing.xml

```diff
@@ -99,7 +99,10 @@
         should ideally execute each line in the code, code coverage should be 100%. To generate the
         JaCoCo report, run the Maven command:
         <br/>
-        <code>mvn clean test jacoco:restore-instrumented-classes jacoco:report@default-report</code>
+        <code>
+          ./mvnw clean test jacoco:restore-instrumented-classes
+          jacoco:report@default-report
```

Comment:
I think, if we make it multiline ,we need to add `\` at the end of first line

Reply:
![image](https://github.com/user-attachments/assets/65e7ea6d-c6e9-40b6-9a30-1cf4b5e83994)

Reply:
single line, so no changes are required.